packages/yew/tests/suspense.rs

```diff
@@ -217,7 +217,7 @@ async fn suspense_not_suspended_at_start() {
 
         Ok(html! {
             <div class="content-area">
-                <textarea value={value.to_string()} oninput={on_text_input}></textarea>
+                <textarea oninput={on_text_input}>{value.to_string()}</textarea>
```

Comment:
@futursolo this test fails after I replaced `value` with children. `<textarea>` is not supposed to take `value attribute. See [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea).

Do you have idea what might be happening?

Reply:
The old implementation uses `set_value` to set value of a textarea which works.

I think if you make the text a child node, then Yew will attempt to appendChild and textarea cannot have childNodes.

Reply:
The value was never being used anywhere, nor was it being updated so I set it to empty string.